R/condense_control.R

```diff
@@ -20,16 +24,17 @@
 #'
 #' ctrl <- condense_control(ctrl, control_parsnip())
 #' str(ctrl)
-condense_control <- function(x, ref) {
+condense_control <- function(x, ref, call = rlang::caller_env()) {
```

Comment:
```suggestion
condense_control <- function(x, ref, ..., call = rlang::caller_env()) {
  check_dots_empty()
```

My reflex here would be to prevent folks from passing `call` by position: